package/kernel/linux/modules/leds.mk

```diff
@@ -24,6 +24,24 @@ $(eval $(call KernelPackage,leds-gpio))
 
 LED_TRIGGER_DIR=$(LINUX_DIR)/drivers/leds/trigger
 
+define KernelPackage/led-group-multi-color
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=LEDs group multi-color support
+  KCONFIG:=CONFIG_LEDS_CLASS_MULTICOLOR \
```

Comment:
No need to select `CONFIG_LEDS_CLASS_MULTICOLOR` here it is already selected in `target/linux/generic/config-6.6`.

Reply:
I removed that and it fixed building on the x86 platform I don't understand why.